portfolio/src/main/java/com/google/sps/servlets/LoginStatusServlet.java

```diff
@@ -0,0 +1,41 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import com.google.gson.Gson;
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+
+import com.google.sps.data.LoginStatus;
+
+
+@WebServlet("/loginStatus")
+public class LoginStatusServlet extends HttpServlet {
+
+    private static final UserService userService = UserServiceFactory.getUserService();
+    private static final Gson gson = new Gson();
+    private static final String COMMENT_HTML = "/comments.html";
```

Comment:
Recommend renaming this to LOGIN_REDIRECT to indicate to the reader how this constant is used.